src/aklite_client_ext.cc

```diff
@@ -282,3 +283,45 @@ bool AkliteClientExt::RebootIfRequired() {
 
   return false;
 }
+
+InstallResult AkliteClientExt::Rollback(const LocalUpdateSource* local_update_source) {
+  auto current = GetCurrent();
+  auto pending_target = GetPendingTarget();
+  auto installation_in_progress = !pending_target.IsUnknown();
+  auto bad_target = installation_in_progress ? pending_target : current;
+
+  LOG_DEBUG << "User initiated rollback. Current Target is " << current.Name();
+  if (installation_in_progress) {
+    LOG_DEBUG << "There is an installation in progress of Target " << pending_target.Name();
```

Comment:
Maybe `Target installation is in progress: <target name>` or `There is Target installation in progress: <target name>`?